crates/re_viewer/src/ui/view_tensor/ui.rs

```diff
@@ -340,6 +323,13 @@ fn tensor_ui(
                 ui.label(ctx.re_ui.error_text(err.to_string()));
             }
         },
+
+        dtype @ TensorDataType::F16 => {
+            ui.label(
+                ctx.re_ui
+                    .error_text(format!("Unsupported tensor type. {dtype}")),
+            );
```

Comment:
what is blocking `f16` support? please create/find an issue for it and link to it here and the other `TODO(854) re-enable once we have F16 tensors`

Reply:
#854 is the F16 issue

Reply:
Link to it from the code please!

Reply:
That's the 854 in the comment... do you always want to write those out as: `https://github.com/rerun-io/rerun/issues/854` `TODO(#854)`